README.md

```diff
@@ -37,3 +31,23 @@ These are features I want to implement if this project gets some stars/tractions
 - [ ] Separate mode where user can guess the district by shape, location and maybe some facts
 - [ ] Better Error Handling
 - [ ] Multiple spelling support
+
+### Contributing Guide:
+
+Want to contribute? Please check the guidelines below:
+*Please use dev branch rather than main for the latest development changes.*
+
+- Fork the repository
+- Create a feature/patch branch from dev branch
```

Comment:
Let's eliminate the `dev` branch and allow user to create PR against `main` branch. This will be really easy to maintain and don't have to wait for someone to merge again in `main` even after the PR is merged to `dev` branch.